common/common.go

```diff
@@ -0,0 +1,14 @@
+package common
+
+const (
+	// FullPersisterDBMode defines the db mode where each event will be saved
+	// to cache and persister
+	FullPersisterDBMode = "full-persister"
+
+	// ImportDBMode defines the db mode where the events will be saved only to cache
+	ImportDBMode = "import-db"
+
+	// OptimizedPersisterDBMode defines the db mode where the events will be saved to
+	// cache, and they will be dumped to persister when necessary
+	OptimizedPersisterDBMode = "optimized-persister"
```

Comment:
nit: I usually like to do a type alias on things like this, to sort of emulate an enum.

```
type DBMode string
FullPersisterDBMode DBMode = "full-persister"
```

Reply:
added db mode type